src/Sylius/Bundle/ApiBundle/Filter/ShippingMethodFilter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Filter;
+
+use ApiPlatform\Core\Api\FilterInterface;
+
+final class ShippingMethodFilter implements FilterInterface
+{
+    public function getDescription(string $resourceClass): array
+    {
+        return [
+            'shipmentId' => [
+                'type' => 'string',
+                'required' => false,
+                'property' => null,
+                'swagger' => [
+                    'name' => 'Shipment',
+                    'description' => 'Id of shipment for which you would like to check available shipping methods',
+                ],
+            ],
+            'tokenValue' => [
+                'type' => 'string',
+                'required' => false,
+                'property' => null,
+                'swagger' => [
+                    'name' => 'Order',
+                    'description' => 'Id of order for which you would like to check available shipping methods. Should be set if you are checking shipping methods for shipment, that is not assigned to user.',
```

Comment:
That's not true, you are using token value of an order, not its id:
```suggestion
                    'description' => 'Token of an order for which you would like to check available shipping methods. Should be set if you are checking shipping methods for shipment, that is not assigned to user.',
```